tutorials/3d/portals/introduction.rst

```diff
@@ -0,0 +1,33 @@
+.. _doc_portals_introduction:
+
+Introduction
+============
+The rooms and portals system is an optional component of Godot that allows you to partition your 3D game levels into a series of :ref:`Room<class_Room>` s (*aka cells*), and :ref:`Portal<class_Portal>` s. Portals are openings between the rooms that the :ref:`Camera<class_Camera>` (and lights) can see through.
+ 
+This allows several features:
+
+- **Portal occlusion culling**, which can increase performance by reducing the number of objects that are drawn, both to cameras and to shadow maps.
+
+- **Gameplay callbacks**, which allow turning off activity outside the gameplay area - AI, physics, animation, processing etc.
+
+The trade off for these features is that we have to manually partition our level into rooms, and add portals between them.
+
+.. note:: Godot portals should not be confused with those in the game of the same name. They do not warp space, they simply represent a window that the camera (or lights) can see through.
+
+Minimizing manual labour
+^^^^^^^^^^^^^^^^^^^^^^^^
+Bear in mind with portalling that although the effort involved in creating rooms for a large level may seem daunting (particularly if you are a one person team!) there are several factors which can make this much easier:
+
+- If you are 'kit bashing' and reusing rooms or areas already, this is an ideal way to save effort. Your level tiles can be rooms, with portals already placed.
+- If you are creating procedural levels, you can create rooms and portals as part of the algorithm.
+- And finally if you are manually creating freeform levels, bear in mind there are absolutely no rules as to how far you go with portalling. Even if you separate a large game level into only two rooms, with a single portal between them, this can give a relatively large performance gain.
```

Comment:
```suggestion
- If you are "kit bashing" and reusing rooms or areas already, this is an ideal way to save effort. Your level tiles can be rooms, with portals already placed.
- If you are creating procedural levels, you can create rooms and portals as part of the procedural generation algorithm you're writing.
- Finally, if you are manually creating freeform levels, bear in mind there are absolutely no rules as to how far you go with portalling. Even if you separate a large game level into only two rooms, with a single portal between them, this can still result in relatively large performance gains.
```